device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/0/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet0      72,73,74,75       Ethernet1/1  1      Ext        100000      Eth0-ASIC0         1           1               8
+Ethernet8      80,81,82,83       Ethernet2/1  2      Ext        100000      Eth8-ASIC0         1           3               8
```

Comment:
Why is the `corePortId`  not continous here ?

Reply:
Fixed